src/core/mac/sub_mac.hpp

```diff
@@ -534,6 +534,12 @@ class SubMac : public InstanceLocator, private NonCopyable
     static void HandleCslTimer(Timer &aTimer);
     void        HandleCslTimer(void);
     void        GetCslWindowEdges(uint32_t &aAhead, uint32_t &aAfter);
+#if OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE
+    // Length of the logging string for CSL debug.
+    static constexpr uint8_t kLogStringSize = 72;
```

Comment:
nit: do we envision this being used elsewhere? If not, we can put this at the top of `SubMac::LogReceived()`.